README.md

```diff
@@ -101,6 +101,7 @@ for its support details.
 |        | Dynamically provision, de-provision File mode ROX volume  | Alpha          | >= v3.0.0          | >= v1.0.0        | Pacific (>=v16.2.0)  | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode RWOP volume | Alpha          | >= v3.5.0          | >= v1.5.0        | Pacific (>=v16.2.0)  | >= v1.22.0         |
 |        | Creating and deleting snapshot                            | GA             | >= v3.1.0          | >= v1.0.0        | Pacific (>=v16.2.0)  | >= v1.17.0         |
+|        | Creating and deleting volume group snapshot               | Alpha          | >= v3.11.0         | >= v1.9.0       | Squid   (>=v19.0.0)  | >= v1.31.0         |
```

Comment:
We don't have support for RBD volume group snapshot yet ? 
Should we only mention CephFS here ?

Reply:
This is already in cephfs section, we have different section for rbd

Reply:
Sorry, Missed it